tests/api2/test_011_user.py

```diff
@@ -93,7 +99,27 @@ def test_03_verify_post_user_do_not_leak_password_in_middleware_log(request):
 
 def test_04_look_user_is_created(request):
     depends(request, ["user_02", "user_01"])
-    assert len(GET('/user?username=testuser').json()) == 1
+    results = GET('/user?username=testuser')
+    assert len(results.json()) == 1, results.text
+
+    u = results.json()[0]
+    g = u['group']
+    to_check = [
+        {
+            'file': '/etc/shadow',
+            'value': f'{u["username"]}:{u["unixhash"]}:18397:0:99999:7:::'
```

Comment:
It's better to hardcode the values that we control. What if `username` field is somehow saved/retrieved incorrectly?